internal/cli/init.go

```diff
@@ -157,8 +163,32 @@ func (c *InitCommand) Run(args []string) int {
 
 	// If we have no config, initialize a new one.
 	if path == "" {
-		if !c.initNew() {
-			return 1
+		proceed, err := c.ui.Input(&terminal.Input{
+			Prompt: "Do you want help generating a waypoint.hcl file? Type 'yes' to initialize the interactive generator or 'no' to generate a template waypoint.hcl file: ",
+			Style:  "",
+			Secret: false,
+		})
+		if err != nil {
```

Comment:
Should we return an error here?

Reply:
Yes I think we do want to return an error there.